Shemshey maxim dao task

ShemsheyMaxim opened the pull request.

ShemsheyMaxim committed:
first commit

ShemsheyMaxim committed:
deleted classes HelloWorld.java, HelloWorldTest.java.
added package model with class Bet, User.
added package dao with interface UserDao, BetDao and class UserDaoImpl, BetDaoImpl.
added package db with class Storage.
added package controller with class ConsoleHandlerImpl.
added class Main.

ShemsheyMaxim committed:
added checkstyle.

ShemsheyMaxim committed:
added modifier access for field in class User

ShemsheyMaxim committed:
added show bet which print user

dmytro-hryhoruk commented on src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java:
```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class ConsoleHandlerImpl {
+    private final BetDao betDao = new BetDaoImpl();
```
other Dao?

ShemsheyMaxim replied:
done

dmytro-hryhoruk commented on src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java:
```diff
@@ -0,0 +1,37 @@
+
+            } catch (ArrayIndexOutOfBoundsException | NumberFormatException e) {
+                System.out.println("Пожалуйста, введите валидные данные: "
+                        + "сумму(число) и коеффициент(число) через пробел для для вашей ставки");
```
Let's write in English)

ShemsheyMaxim replied:
done

dmytro-hryhoruk commented on src/main/java/core/basesyntax/controller/ConsoleHandlerImpl.java:
```diff
@@ -0,0 +1,37 @@
+                continue;
+            }
+            betDao.add(bet);
+            System.out.println(bet);
+            System.out.println("Введите ещё одну ставку или введите \"стоп\" "
+                    + "чтобы завершить работу");
```
Let's pring all bets using getAll() after loop

ShemsheyMaxim replied:
done

ShemsheyMaxim committed:
added Dao for User in class ConsoleHandlerImpl,
added method handleUser() in class ConsoleHandlerImpl,
renamed method handle to handleBet.

ShemsheyMaxim committed:
added package exception and created class AnnotationException,
added annotation Dao for BetDaoImpl and UserDaoImpl classes,
added annotation Inject for ConsoleHandlerImpl class,
added package factory and created class Factory,
created annotations Dao and Inject,
created and realised class Injector.

ShemsheyMaxim committed:
added method checkObjectOnAnnotationDao in class Injector

ShemsheyMaxim commented:
I forgot merge pull request after first part.

RomanDubovskyi commented:
you have files from prev hw, they should not be in this PR, fix it before review

ShemsheyMaxim committed:
Revert "added method checkObjectOnAnnotationDao in class Injector"

This reverts commit e437b95a

ShemsheyMaxim committed:
Revert "added package exception and created class AnnotationException, added annotation Dao for BetDaoImpl and UserDaoImpl classes, added annotation Inject for ConsoleHandlerImpl class, added package factory and created class Factory, created annotations Dao and Inject, created and realised class Injector."

This reverts commit a7c02cbd
